### DISCUSSION TITLE
Deleted and recovered plots are not recognized

### DISCUSSION THREAD

**Discussion by DIlbert25** [Chia-Network/chia-blockchain#5718] [General]
I accidentally deleted 11 plots.

I used the EaseUS Data Recovery program and recovered the 11 deleted files.

But now the Chia app doesn't recognize these plots.

The "plots check" command displays the following message in these files:

"Invalid plot header magic Traceback (most recent call last):"

"2021-05-20T15: 51: 46.030 chia.plotting.check_plots: WARNING 11 invalid plots found:"

Does anyone have any ideas to help, or a tip from another program to recover these files?

**Comment by DataWizard0103** [comment]
If you're facing issues with recovered Chia plot files, you may want to try using a different data recovery software. These tools offer various recovery methods that could result in more successful recovery of damaged plot files. Before attempting any recovery or repair procedures, make sure the recovered data is undamaged. If none of the recovery methods work, you may need to contact Chia support or consider re-plotting the files. Be cautious and remember to backup critical information to avoid further loss.

**Comment by BrandtH22** [comment]
Hey @DIlbert25 since it is only 11 plots it might make more sense to just replot then try and recover the previous ones. 
Advancements in plotting software has brought plotting down to as little as 2 minutes or less with the right hardware.

More information about the plotting software can be found here: https://docs.chia.net/plotting-software/

Feel free to jump into our discord server for more thorough and timely support:
https://discord.gg/chia
